src/ctia/entity/attack_pattern.clj

```diff
@@ -48,10 +48,7 @@
 (def realize-attack-pattern
   (default-realize-fn "attack-pattern" NewAttackPattern StoredAttackPattern))
 
-(def attack-pattern-fields
-  (concat sorting/base-entity-sort-fields
-          sorting/sourcable-entity-sort-fields
-          [:name]))
+(def attack-pattern-fields sorting/default-entity-sort-fields)
```

Comment:
remove `:name` and add describable-fields correspond to [`default-entity-sort-fields`](https://github.com/threatgrid/ctia/blob/ac300dfa9a86cb61f49e8a490da0982e7164f234/src/ctia/schemas/sorting.clj#L15):
``` clojure
(def default-entity-sort-fields
  (concat base-entity-sort-fields
          sourcable-entity-sort-fields
          describable-entity-sort-fields))
```